src/main/scala/higherkindness/rules_scala/workers/zinc/test/TestRunner.scala

```diff
@@ -46,6 +46,9 @@ object TestRunner {
       .choices("HIGH", "MEDIUM", "LOW")
       .setDefault_("MEDIUM")
     parser
+      .addArgument("--options")
```

Comment:
I'm wondering if a better name might be `--framework_args`, `--test_framework_args`, or `--test_args`